dom.bs

```diff
@@ -3398,6 +3398,19 @@ dictionary MutationObserverInit {
    <a>attribute</a> mutations need to be
    observed and {{MutationObserverInit/attributes}} is true
    or omitted.
+
+   <dt>{{MutationObserverInit/elementHasAttributeFilter}}
```

Comment:
We are filtering based on just attribute names, right? So, shouldn't it just be `elementHasAttributeNameFilter` ?